lib/services/github_issues.rb

```diff
@@ -256,6 +256,8 @@ def issue_body(resource)
     if resource.description.attachments.present?
       issue_body_parts << attachments_in_body(resource.description.attachments)
     end
+    issue_body_parts << "Created from Aha! #{resource.url}" if resource.respond_to?(:url)
```

Comment:
It looks like updates in github are ignored on the aha side currently. The update command to github rewrites the issue body.